webapp/src/components/CalendarPage.tsx

```diff
@@ -1,9 +1,17 @@
+import {
+  Calendar,
+  luxonLocalizer,
+  Event as RBCEvent,
+} from 'react-big-calendar';
 import { Container } from '@mui/material';
+import { DateTime, Settings } from 'luxon';
+import { ProgramWithActivities } from '../types/api';
 import React from 'react';
+import 'react-big-calendar/lib/css/react-big-calendar.css';
 import useApiData from '../helpers/useApiData';
 
-import { ProgramWithActivities } from '../types/api';
-import Program from './Program';
+Settings.defaultZone = 'America/Los_Angeles';
```

Comment:
Perhaps we should set this to "America/Vancouver"?

Reply:
I will do it!